packages/babel-helpers/src/helpers.js

```diff
@@ -5,64 +5,75 @@ import template from "babel-template";
 const helpers = {};
 export default helpers;
 
-helpers.typeof = template(`
-  (typeof Symbol === "function" && typeof Symbol.iterator === "symbol")
-    ? function (obj) { return typeof obj; }
-    : function (obj) {
+function defineHelper(str) {
+  return template(str, { sourceType: "module" });
+}
+
+helpers.typeof = defineHelper(`
+  export default function _typeof(obj) {
+    if (typeof Symbol === "function" && typeof Symbol.iterator === "symbol") {
+      _typeof = function (obj) { return typeof obj; };
+    } else {
+      _typeof = function (obj) {
         return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype
           ? "symbol"
           : typeof obj;
       };
+    }
+
+    return _typeof(obj);
+  }
 `);
 
-helpers.jsx = template(`
-  (function () {
-    var REACT_ELEMENT_TYPE = (typeof Symbol === "function" && Symbol.for && Symbol.for("react.element")) || 0xeac7;
+helpers.jsx = defineHelper(`
+  var REACT_ELEMENT_TYPE;
 
-    return function createRawReactElement (type, props, key, children) {
-      var defaultProps = type && type.defaultProps;
-      var childrenLength = arguments.length - 3;
+  export default function _createRawReactElement(type, props, key, children) {
```

Comment:
I did this so things were less likely to need to be renamed, and I still kind of feel like that is good? Especially since helpers can have multiple functions now, it's nice to make collision less likely so things keep their predefined name.